@kiva/kv-components/vue/KvInlineActivityCard.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+	<div
+		class="tw-flex tw-items-center tw-gap-1 tw-py-0.5 tw-px-1 tw-shadow-lg"
+		style="border-radius: 8px;"
```

Comment:
Is there a tailwind class that's close enough?

Reply:
Or change to 4px, could make more sense for that card.

Reply:
If 4px seems to make sense, go for it. We can generally use our existing styles in Tailwind.

Reply:
Very biased sometimes...